docs/usage/.pages

```diff
@@ -1,5 +1,6 @@
 nav:
   - Home: 'index.md'
+  - Bot comparison: 'bot-comparison.md'
```

Comment:
We'll need to decide _where_ to put this information in our docs sidebar.

I think it should go somewhere really visible (like the second page) or we'll still get questions like: "Do you guys have a comparison between Dependabot and Renovate"... 🙃

Reply:
I would call the page "Renovate vs ...", that way we can add for example Snyk

Reply:
I agree we should have some generic name that can fit an expanded comparison. But I think we want to stay away from the vs word though to keep things fair? 🙂

Adding Snyk is something we can do later, let's focus on the Dependabot comparison in this PR.